lavamoat/browserify/flask/policy.json

```diff
@@ -997,7 +982,7 @@
         "@metamask/eth-json-rpc-middleware>clone": true,
         "@metamask/eth-json-rpc-middleware>pify": true,
         "@metamask/eth-json-rpc-middleware>safe-stable-stringify": true,
-        "@metamask/eth-snap-keyring>@metamask/eth-sig-util": true,
+        "@metamask/eth-trezor-keyring>@metamask/eth-sig-util": true,
         "eth-rpc-errors": true,
```

Comment:
@legobeat why are these changes occurring when updating the policy files? This PR should have any impact in the trezor keyring

Reply:
Thanks for the explanation @legobeat